### DISCUSSION TITLE
ProgramData vs Roaming for Multi-User Environment

### DISCUSSION THREAD

**Discussion by Helium100** [seerge/g-helper#3231] [Q&A] [answered]
Hello

I have a small concern and thought it might be worth asking for your perspective.

I'm using a PC with multiple user accounts, and I'm facing a situation where all users want to use GHelper. However, by default, each user has a different config.json file in their Roaming folder.

To address this, I'm planning to create a symlink for each user's GHelper Roaming folder, pointing it to a location in C:/ProgramData, which I believe is better suited for shared data among multiple users. While doing this, I also intend to restrict write access to certain users only.

I hope this solution will work.

However, I'm still curious—why does GHelper prefer to use separate config files for each user instead of a global config file?

Thanks!

**Comment by seerge** [comment, marked as answer]
@Helium100 hello,

Actually I think it's good when 2 different users can have their own settings for G-Helper :) 

But if you want to use same config for everyone  just place `config.json` into same folder as `GHelper.exe` (and start app again after). Then it will just use that config instead of one in `%appdata` folder.

**Reply by Helium100** [reply]
Thank you seerge!

I'm going to test your solution first.

The most challenging part will be reminding the children not to tamper with the fan and power settings, but should not be impossible! :)

Have a great day!

**Reply by seerge** [reply]
@Helium100 keep in mind that G-Helper's startup task in Task Scheduler is also user-specific (as app launches after user logon).

So you will probably have to create multiple startup tasks for each user then.

**Reply by Helium100** [reply]
I will keep that in mind then, thank you!
